### PR TITLE
Login and Products Page

### PR DESCRIPTION
**Pull request by bhargaviRdy**
Completed Login and Products Page with Filters and responsiveness
Look into https://vibe-store-bhargavi.netlify.app/login.html and https://vibe-store-bhargavi.netlify.app/productpage.html for changes
( Note: index.html will be modified later )
image

### CODE REVIEW

File: login.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login | VibeStore</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css"
+        integrity="sha512-HK5fgLBL+xu6dm/Ii3z4xhlSUyZgTT9tuc/hSrtw6uzJOvgRr2a9jyxxT1ely+B+xFAmJKVSTbpM/CuL7qxO8w=="
+        crossorigin="anonymous" />
+    <link rel="stylesheet" href="https://vibe-ui-bhargavi.netlify.app/components/vibeui.css">
+    <link rel="stylesheet" href="./css/authentication.css">
+</head>
+<body>
+    <div class="nav-bar p-3">
```
Comment (ShreyashMahajan) [inline login.html]: try to use semantic tag for navbar 'nav'.

File: login.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        <ul class="ul-none navbar--icons">
+            <li class="list-inline-item navbar--icon px-2">
+                <a href="" class="btn"><i class="fas fa-user"></i></a></li>
+            <li class="list-inline-item navbar--icon px-2">
+                <a href="" class="btn"><i class="fas fa-heart"></i></a></li>
+            <li class="list-inline-item navbar--icon px-2">
+                <a href="" class="btn"><i class="fas fa-shopping-cart"></i></a></li>
+        </ul>
+    </div>
+    <div class="main">
```
Comment (ShreyashMahajan) [inline login.html]: same here 'main'

**Review by ShreyashMahajan — APPROVED** [review]
The naming convention is great. UI is also very good and fully responsive. Good to go.

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (harshitadaswani) [inline .vscode/settings.json:3]: add this file to git ignore

File: login.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+            <input class="searchbar-input" type="text" name="search" id="navbar-search" placeholder="Search">
+        </div>
+    
+        <ul class="ul-none navbar--icons">
+            <li class="list-inline-item navbar--icon px-2">
+                <a href="" class="btn"><i class="fas fa-user"></i></a></li>
+            <li class="list-inline-item navbar--icon px-2">
+                <a href="" class="btn"><i class="fas fa-heart"></i></a></li>
+            <li class="list-inline-item navbar--icon px-2">
+                <a href="" class="btn"><i class="fas fa-shopping-cart"></i></a></li>
```
Comment (harshitadaswani) [inline login.html]: give values to `href` attributes

**Review by harshitadaswani — APPROVED** [review]
Refactor given suggestions and go for the merge

File: login.html (hunk 1 above)
Comment (bhargaviRdy) [reply login.html]: thanks for the suggestion, will make the changes

File: .vscode/settings.json (hunk 3 above)
Comment (bhargaviRdy) [reply .vscode/settings.json:3]: sure, will do

**Pull request merged**